pages/Addons.md

```diff
@@ -72,6 +72,7 @@ DankTech2 | Sefiraat | <details><summary>expand</summary>DankTech is a plugin th
 Networks | Sefiraat | <details><summary>expand</summary>Networks is an addon that aims to provide a AE2/Refined Storage style storage solution in a straight-forward but powerful manner. The addon works alongside core-slimefun cargo to allow you to insert mass storage along side your current automation with ease</details> | [Wiki](https://github.com/Sefiraat/Networks/wiki) | [GitHub](https://github.com/Sefiraat/Networks) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/Networks/master/)
 SfChunkInfo | FN_FAL113 | <details><summary>expand</summary>Scan the amount of slimefun blocks in a chunk using a single command.</details> | [Readme](https://github.com/FN-FAL113/SfChunkInfo/blob/main/README.md) | [GitHub](https://github.com/FN-FAL113/SfChunkInfo) | [Builds](https://thebusybiscuit.github.io/builds/FN-FAL113/SfChunkInfo/main/)
 VillagerUtil | Apeiros-46B | <details><summary>expand</summary>VillagerUtil adds a few tools that allow for easy transportation of villagers, curing zombified villagers, cycling trades and more.</details> | [Readme](https://github.com/Apeiros-46B/VillagerUtil/blob/main/README.md) | [GitHub](https://github.com/Apeiros-46B/VillagerUtil) | [Builds](https://thebusybiscuit.github.io/builds/Apeiros-46B/VillagerUtil/main)
+MissileWarfare | Kaiyan2006 | <details><summary>expand</summary>Missile Warfare adds a bunch of missiles to slime fun that can be fired at others!</details> | [GitHub](https://github.com/koiboi-dev/MissileWarfare) | N/A | [Releases] (https://github.com/koiboi-dev/MissileWarfare/releases)
```

Comment:
Is the addon called MissileWarfare or Missile Warfare - you call it both in here - one or the other I think :)